.hintrc

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
What is this file for?

Reply:
Not exactly sure what the file is and when I added it, but I have deleted it, ran the application, and had no issues

Reply:
Thanks!